schema/test/unit/util_functions/import_swrs_operators_test.sql

```diff
@@ -51,7 +51,7 @@ select results_eq(
       (new_form_data->'swrsOrganisationId')::int,
       operation::text,
       change_status::text
-    from cif.form_change where form_data_table_name='operator'
+    from cif.form_change where form_data_table_name='operator' order by id
```

Comment:
I wonder why the changes affected this test

Reply:
> If sorting is not chosen, the rows will be returned in an unspecified order. The actual order in that case will depend on the scan and join plan types and the order on disk, but it must not be relied on. A particular output ordering can only be guaranteed if the sort step is explicitly chosen.

Not sure what exactly happened but it's something we can't rely on, best practice should be to include an `order by` clause.